lib/features/navigation_tab_view/navigation_tab_view.dart

```diff
@@ -0,0 +1,109 @@
+import "package:auto_route/auto_route.dart";
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../config/ui_config.dart";
+import "../../theme/app_theme.dart";
+import "../../utils/context_extensions.dart";
+import "../analytics/show_feedback_tile.dart";
+import "../bottom_nav_bar/bottom_nav_bar_icon_icons.icons.dart";
+import "../home_view/widgets/logo_app_bar.dart";
+import "../navigator/utils/navigation_commands.dart";
+import "widgets/about_the_app_tile.dart";
+import "widgets/about_us_tile.dart";
+import "widgets/navigation_tile.dart";
+import "widgets/notification_button.dart";
+import "widgets/small_tile.dart";
+
+@RoutePage()
+class NavigationTabView extends ConsumerWidget {
+  const NavigationTabView({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final children = [
+      const SizedBox(height: NavigationTabViewConfig.universalPadding),
+      const AboutUsTile(),
+      const ShowFeedbackTile(),
+      const SizedBox(height: NavigationTabViewConfig.universalPadding / 3),
+      _NavigationRow(
+        child1: SmallTileCard(
+          onTap: ref.navigateDepartments,
+          title: context.localize.departments,
+          icon: const Icon(BottomNavBarIcon.faculty_icon, size: 30),
+        ),
+        child2: SmallTileCard(
+          onTap: () {
+            // TODO(simon-the-shark): implement review request
+          },
+          title: context.localize.leave_a_review,
+          icon: const Icon(Icons.star, color: Color(0xFFFFDF00), size: 30),
```

Comment:
WDYT about making this color available in our color scheme? maybe will be reusable somewhere in the future. Imo maybe it could be darker a little? For better visibility

Reply:
Sure, I'll try